lib/assets/javascripts/turbograft/remote.coffee

```diff
@@ -99,6 +99,22 @@ class TurboGraft.Remote
         if (radioOrCheck && input.checked) || !radioOrCheck
           callback(input)
 
+  _enabledInputs: (form) ->
+    selector = "input:not([type='reset']):not([type='button']):not([type='submit']):not([type='image']), select, textarea"
+    inputs = Array.prototype.slice.call(form.querySelectorAll(selector))
```

Comment:
`<something>.prototype` can be condensed to `<something>::` in CS, so you can write this a little more succinctly with `Array::slice.call(form.querySelectorAll(selector))`